pom.xml

```diff
@@ -188,6 +188,7 @@
     <module>common</module>
     <module>dora</module>
     <module>webui</module>
+    <module>native</module>
```

Comment:
sort?

Reply:
I move the module to a profile to avoid blocking a lot build pipelines